# Generated by Django 4.0.1 on 2022-02-04 13:33

import books.validators
import django.core.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('books', '0004_alter_book_publisheddate'),
    ]

    operations = [
        migrations.AlterField(
            model_name='book',
            name='ISBN_10',
            field=models.CharField(blank=True, max_length=10, null=True, validators=[django.core.validators.MinLengthValidator(10, message='Incorrect data - 10 characters are needed.'), django.core.validators.MaxLengthValidator(10, message='Incorrect data - 10 characters are needed.')]),
        ),
        migrations.AlterField(
            model_name='book',
            name='ISBN_13',
            field=models.CharField(blank=True, max_length=13, null=True, validators=[django.core.validators.MinLengthValidator(13, message='Incorrect data - 13 characters are needed.'), django.core.validators.MaxLengthValidator(13, message='Incorrect data - 13 characters are needed.')]),
        ),
        migrations.AlterField(
            model_name='book',
            name='canonicalVolumeLink',
            field=models.URLField(blank=True, null=True),
        ),
        migrations.AlterField(
            model_name='book',
            name='pageCount',
            field=models.CharField(blank=True, max_length=6, null=True, validators=[books.validators.validate_isnumeric]),
        ),
        migrations.AlterField(
            model_name='book',
            name='publishedDate',
            field=models.IntegerField(validators=[django.core.validators.MinValueValidator(-3000, message='Impossible year'), django.core.validators.MaxValueValidator(2023, message='Impossible year')]),
        ),
    ]
